lib/OpenQA/Utils.pm

```diff
@@ -361,33 +342,20 @@ sub run_cmd_with_log_return_error ($cmd, %args) {
     }
 }
 
-sub asset_type_from_setting {
-    # passing $value is optional but makes the result more accurate
-    # (it affects only UEFI_PFLASH_VARS currently).
-    my ($setting, $value) = @_;
-    $value //= '';
-    if ($setting eq 'ISO' || $setting =~ /^ISO_\d+$/) {
-        return 'iso';
-    }
-    if ($setting =~ /^HDD_\d+$/) {
-        return 'hdd';
-    }
+# passing $value is optional but makes the result more accurate
+# (it affects only UEFI_PFLASH_VARS currently).
+sub asset_type_from_setting ($setting, $value = '') {
+    return 'iso' if $setting eq 'ISO' || $setting =~ /^ISO_\d+$/;
+    return 'hdd' if $setting =~ /^HDD_\d+$/;
     # non-absolute-path value of UEFI_PFLASH_VARS treated as HDD asset
-    if ($setting eq 'UEFI_PFLASH_VARS' && $value !~ m,^/,) {
-        return 'hdd';
-    }
-    if ($setting =~ /^REPO_\d+$/) {
-        return 'repo';
-    }
-    if ($setting =~ /^ASSET_\d+$/ || $setting eq 'KERNEL' || $setting eq 'INITRD') {
-        return 'other';
-    }
+    return 'hdd' if $setting eq 'UEFI_PFLASH_VARS' && $value !~ m,^/,;
```

Comment:
```suggestion
    # non-absolute-path value of UEFI_PFLASH_VARS treated as HDD asset
    return 'hdd' if $setting =~ /^HDD_\d+$/ || ($setting eq 'UEFI_PFLASH_VARS' && $value !~ m,^/,);
```
The preview doesnt include the line 349 above but this is what I meant

Reply:
then the comment wouldn't comment the right statement anymore. I prefer to keep the old style due to the comment